src/app/api/check-username-unique/route.ts

```diff
@@ -1,38 +1,39 @@
-import dbConnect from "@/lib/dbConnect";
-import UserModel from "@/model/User";
+
 import { z } from "zod";
 import { usernameValidation } from "@/schemas/signUpSchema";
+import db from "@/lib/db";
 
 const UsernameQuerySchema = z.object({
     username: usernameValidation
 });
 
+export const dynamic = 'force-static'
+
 export async function GET(request: Request) {
-    await dbConnect();
+
 
     try {
         const { searchParams } = new URL(request.url);
         const queryParams = {
             username: searchParams.get("username")
         };
-        
+
         const result = UsernameQuerySchema.safeParse(queryParams);
-        
+
         if (!result.success) {
             const usernameErrors = result.error.format().username?._errors;
             return new Response(JSON.stringify({
                 success: false,
                 message: `${usernameErrors}`
             }), { status: 400 });
         }
-        
+
```

Comment:
I think here some error occurred while checking unique username

Reply:
I fixed that by added 'force-dynamic'